Formula/akamai.rb

```diff
@@ -0,0 +1,29 @@
+class Akamai < Formula
+  desc "CLI toolkit for working with Akamai's API from the command-line."
+  homepage "https://github.com/akamai/cli"
+  url "https://github.com/akamai/cli/archive/0.1.0.tar.gz"
+  sha256 "68a49db1c397ff55185b06252d1f2590395c6193566cdb52b2c910dd7d6c3889"
+
+  version "0.1.0"
```

Comment:
This will trigger an audit failure since we can actually successfully detect that version from the url.